people/ondrej-bojar.md

```diff
@@ -0,0 +1,12 @@
+---
+grand_parent: Community
+parent: People
+title: Ondřej Bojar
+description: Machine translation researcher
+---
+
+**Ondřej Bojar** is a researcher and associate professor at the [Institute of Formal and Applied Linguistics](https://ufal.mff.cuni.cz/) at Charles University in Prague, widely cited for his work on statistical and later [neural machine translation](/neural-machine-translation).
```

Comment:
I would separate this sentence into two:

> He is widely cited for his work on [statistical](/approaches/statistical-machine-translation.md) and [neural machine translation](/approaches/neural-machine-translation.md).

Reply:
Good point.